src/buffer_1.rs

```diff
@@ -1,15 +1,17 @@
+//! `buffer_1.rs`: an 8kB thread-unsafe static buffer used for various things
+//!
+//! TODO: document those things and find a better way to handle them
+
 use buffer::{self, Buffer};
 
-#[no_mangle]
-pub static mut buffer_1_space: [u8; 8192] = [0u8; 8192];
+const BUFFER_1_SIZE: usize = 8192;
+
+static mut BUFFER_1_SPACE: [u8; BUFFER_1_SIZE] = [0u8; BUFFER_1_SIZE];
 
-static mut it: Buffer = Buffer {
-    x: buffer_1_space.as_mut_ptr(),
+pub static mut BUFFER_1: Buffer = Buffer {
+    x: 0 as *mut u8,
```

Comment:
Need a better solution for this. Perhaps we can leverage the `as_mut_ptr()` accessor?